.github/workflows/build.yml

```diff
@@ -0,0 +1,45 @@
+name: Build SaintCoinach
+on: [push, pull_request]
+concurrency: build_sc
+
+jobs:
+  build:
+    name: Build on Windows
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Setup Nuget
+        uses: nuget/setup-nuget@v1
+        with:
+          nuget-version: latest
+      - name: Restore Nuget Packages
+        run: nuget restore SaintCoinach.sln
+      - name: Define VERSION
+        id: define-version
+        run: |
+          $env:COMMIT = $env:GITHUB_SHA.Substring(0, 7)
+          echo "::set-output name=commit::$env:COMMIT"
+      - name: Build DotNet4
+        run: |
+          cd "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\"
+          .\MSBuild.exe $Env:GITHUB_WORKSPACE\SaintCoinach.sln /t:Build /p:Configuration=Release
```

Comment:
```suggestion
      - uses: actions/checkout@v2
        with:
          submodules: recursive

      - name: Define Version
        id: define-version
        run: |
          $env:COMMIT = $env:GITHUB_SHA.Substring(0, 7)
          echo "::set-output name=commit::$env:COMMIT"

      - uses: nuget/setup-nuget@v1
        with:
          nuget-version: latest

      - name: Restore Nuget Packages
        run: nuget restore SaintCoinach.sln

      - name: Build DotNet4
        run: |
          cd "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\"
          .\MSBuild.exe $Env:GITHUB_WORKSPACE\SaintCoinach.sln /t:Build /p:Configuration=Release
```

Reply:
This bit is just so it's easier to read, and busted clones will fail before slow stuff like nuget gets to run.